akka-projection-testkit/src/test/scala/akka/projection/testkit/ProjectionTestKitSpec.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.testkit
+
+import akka.{ Done, NotUsed }
+import akka.actor.ClassicActorSystemProvider
+import akka.actor.testkit.typed.scaladsl.{ ScalaTestWithActorTestKit, TestProbe }
+import akka.projection.Projection
+import akka.stream.{ KillSwitch, KillSwitches }
+import akka.stream.scaladsl.Source
+import akka.stream.scaladsl.{ Keep, Sink }
+import akka.stream.testkit.TestSubscriber
+import akka.stream.testkit.scaladsl.TestSink
+import org.scalatest.wordspec.AnyWordSpecLike
+import akka.actor.typed.scaladsl.adapter._
+
+import scala.concurrent.{ Future, Promise }
+
+class ProjectionTestKitSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike {
+
+  val projectionTestKit: ProjectionTestKit = ProjectionTestKit(testKit)
+
+  "ProjectionTestKit" must {
+
+    "run a projection" in {
```

Comment:
This tests is actually confirming that `ProjectionTestKit.run` is running the projection.